test/object-store/benchmarks/object.cpp

```diff
@@ -328,7 +328,7 @@ TEST_CASE("Benchmark object", "[benchmark]") {
                     {"name", name.str()},
                     {"age", static_cast<int64_t>(i)},
                 };
-                Object::create(d, r, person_schema, Any(person), CreatePolicy::ForceCreate);
+                Object::create(d, r, person_schema, util::Any(person), CreatePolicy::ForceCreate);
```

Comment:
I think good practice going forward is to use `std::any` where possible. Not blocking though because it is aliased to the same thing.